src/NUnitFramework/framework/Internal/Execution/ParallelWorkItemDispatcher.cs

```diff
@@ -306,8 +304,12 @@ internal static ExecutionStrategy GetExecutionStrategy(WorkItem work)
                 work.Test is TestFixture && work.Context.ParallelScope.HasFlag(ParallelScope.Fixtures))
                     return ExecutionStrategy.Parallel;
 
-            // If all else fails, run on same thread
-            return ExecutionStrategy.Direct;
+            // There is no scope specified either on the item itself or in the context.
+            // In that case, simple work items are test cases and just run on the same
+            // thread, while composite work items and teardowns are non-parallel.
+            return work is SimpleWorkItem
+                ? ExecutionStrategy.Direct
+                : ExecutionStrategy.NonParallel;
```

Comment:
An alternative, which I mentioned in a different comment, is to use `work.Test.IsSuite` to make the distinction. I don't particularly like this pushing through to the test, but I've done it in a few places in this method already. My thought is that it will eventually become clear what members are needed in the WorkItem in order to avoid that.